### PR TITLE
Replace Yolo v2 tiny model

### PR DESCRIPTION
**Pull request by eizamaliev**

### CODE REVIEW

**Comment by eaidova** [comment]
@eizamaliev do you check, accuracy metric and AC config is not changed?

**Review by eaidova — APPROVED** [review]

**Review by IRDonch — APPROVED** [review]

File: models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md [hunk 1]
```diff
@@ -2,39 +2,26 @@
...
-    tensorflowjs
-    ```
-1. Download the model from [here](https://github.com/shaqian/tfjs-yolo-demo/tree/master/dist/model/v2tiny) (tested on `aa4354c` commit).
-2. Convert the model to Keras\* format using `tensorflowjs_converter` script, e.g.:
-    ```
-    tensorflowjs_converter --input_format tfjs_layers_model --output_format keras <model_in>.json <model_out>.h5
-    ```
-3. Convert the produced model to protobuf format.
+1. Download or clone the original [repository](https://github.com/david8862/keras-YOLOv3-model-set) (tested on `d38c3d8` commit).
+2. Use the folllowing commands to get original model (named `yolov2_tiny` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/blob/d38c3d865f7190ee9b19a30e91f2b750a31320c1/README.md)  file in the official repository):
```
Comment (IRDonch) [inline models/public/yolo-v2-tiny-tf/yolo-v2-tiny-tf.md:10]:
```suggestion
2. Use the following commands to get original model (named `yolov2_tiny` in repository) and convert it to Keras\* format (see details in the [README.md](https://github.com/david8862/keras-YOLOv3-model-set/blob/d38c3d865f7190ee9b19a30e91f2b750a31320c1/README.md)  file in the official repository):
```
